components/LessonTitleCard.test.js

```diff
@@ -1,7 +1,33 @@
 import React from 'react'
 import LessonTitleCard from './LessonTitleCard'
 import { render, screen, fireEvent, waitFor } from '@testing-library/react'
-describe('LessonTitleCard component', () => {
+
+const mockBack = jest.fn()
+
+jest.mock('next/router', () => ({
+  useRouter: () => ({
+    back: mockBack
+  })
+}))
```

Comment:
We already have a manual mock for the router, don't need to mock it again.

https://github.com/garageScript/c0d3-app/blob/master/__mocks__/next/router.js

You can just use the back method directly and jest will mock it automatically.

Reply:
Removed custom mock, now using the existing named mock